contrib/systemd/README.md

```diff
@@ -5,6 +5,8 @@ author: Ben Boeckel
 date: 2015-03-22
 contributors: Abdo Roig-Maranges
 updated: 2015-03-25
+contributors: benutzer193
+updated: 2017-06-01
```

Comment:
You should add yourself like this:

` contributors: name A, name B`

Those are jekyll headers used for the website.